LDAP-Auth/LDAP-Auth.csproj

```diff
@@ -13,7 +13,7 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Jellyfin.Controller" Version="10.3.*" />
+    <PackageReference Include="Jellyfin.Controller" Version="10.4-*" />

```

Comment:
This won’t work with the actual 10.4 release, right?

Reply:
That's what I'm not sure (see commit message). In any case it's needed for right now and might need to change on release if it doesn't work.

Reply:
My much better TL;DR to @dkanada from chat, just for posterity:

> The alternative is manually specifying the prerelease version, which we'll have to change anyways. So this is basically "it might work, it might not. but if not, we'd have to do the same change anyways. And I'm just hoping it does work